addressing/marketplace_addressing/addresser.py

```diff
@@ -141,5 +141,5 @@ def address_is(address):
 
     elif _contains(infix, OfferSpace):
         return AddressSpace.OFFER
-    else:
-        return AddressSpace.OTHER_FAMILY
+
+    raise ValueError
```

Comment:
This change fixes lint _and_ changes behavior.  Is that the correct course here?

Reply:
The raise ValueError shouldn't ever be hit and would be an exceptional case.

Reply:
I think I will change this one and make sure it doesn't change behavior.

Reply:
The change is fine, just wanted to make sure there was awareness.